### DISCUSSION TITLE
B站番剧区如何退回到原先的版本

### DISCUSSION THREAD

**Discussion by Unknown-shu** [the1812/Bilibili-Evolved#2227] [提问 / Q&A]
灰度到了，失去快捷键以及逐帧体感极差（目前通过H5脚本来代替）
想问问有無退回的方法
![image](https://user-images.githubusercontent.com/64715642/131096110-625b3ad7-b81b-4ac5-b999-bfdf9f80757e.png)

**Comment by Geeyun-JY3** [comment]
等修复：#1992
要不就暂时把脚本相关功能关了，新版播放器这种东西早晚会灰度到，脚本改变不了什么。
